src/test/test_utils.cpp

```diff
@@ -129,6 +129,11 @@ void waitForOVMSResourcesCleanup(ovms::ModelManager& manager) {
 
 bool createConfigFileWithContent(const std::string& content, std::string filename) {
     std::ofstream configFile{filename};
+    // Check if the file was successfully opened
+    if (!configFile.is_open()) {
+        SPDLOG_INFO("Failed to open file: {}", filename);
```

Comment:
I don't think we check everywhere the status of this so throwing may be better idea here.

Reply:
done